### DISCUSSION TITLE
App Thinning support for Xcode archive

### DISCUSSION THREAD

**Discussion by LemonSpike** [fastlane/fastlane#18485] [Ideas]
Xcode enables app thinning for Ad Hoc distributed apps when going through the Organizer. I was wondering, could this option be added to Fastlane for iOS targets? I think this feature could be useful, and couldn't find a solution to this in the current documentation. Let me know your thoughts. 😊

**Comment by kambala-decapitator** [comment]
it's already available, just pass proper params to `gym`'s `export_options` parameter
